src/main/java/org/mastodon/mamut/tomancak/divisiontagset/CellDivisionsTagSetCommand.java

```diff
@@ -0,0 +1,170 @@
+ * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ * #L%
+ */
+package org.mastodon.mamut.tomancak.divisiontagset;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Iterator;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+import org.apache.commons.lang3.tuple.Pair;
+import org.mastodon.mamut.ProjectModel;
+import org.mastodon.mamut.model.Model;
+import org.mastodon.mamut.model.Spot;
+import org.mastodon.mamut.model.branch.BranchSpot;
+import org.mastodon.mamut.model.branch.ModelBranchGraph;
+import org.mastodon.model.tag.TagSetStructure;
+import org.mastodon.util.ColorUtils;
+import org.mastodon.util.TagSetUtils;
+import org.scijava.Cancelable;
+import org.scijava.command.Command;
+import org.scijava.plugin.Parameter;
+import org.scijava.plugin.Plugin;
+import org.scijava.util.ColorRGB;
+
+/**
+ * {@link Command A SciJava command} that is called by the {@link CellDivisionsTagSetPlugin}.
+ * <p>
+ * The command creates a tag set that highlights cell divisions. The user interface allows to
+ * specify the number of spots to highlight before and after a division, and the highlight and
+ * background colors.
+ */
+@Plugin( type = Command.class, label = "Add a Tag Set to Highlight Cell Divisions", visible = false )
+public class CellDivisionsTagSetCommand implements Command, Cancelable
+{
+	@Parameter
+	private ProjectModel projectModel;
+
+	@Parameter( label = "Tag set name" )
+	private String tagSetName = "cell divisions";
+
+	@Parameter( label = "Default color" )
+	private ColorRGB defaultColor = new ColorRGB( "dark gray" );
+
+	@Parameter( label = "Highlight color" )
+	private ColorRGB highlightColor = new ColorRGB( "pink" );
+
+	@Parameter( label = "Timepoints to highlight before and after cell division" )
+	private int n = 3;
+
+	private List< TagSetStructure.Tag > tagsBefore;
+
+	private List< TagSetStructure.Tag > tagsAfter;
+
+	private TagSetStructure.Tag backgroundTag;
+
+	private Model model;
+
+	@Override
+	public void run()
+	{
+		projectModel.getBranchGraphSync().sync();
+		model = projectModel.getModel();
+		createTagSet();
+		applyTags();
+	}
+
+	private void createTagSet()
+	{
+		int defaultColor = this.defaultColor.getARGB();
+		int highlightColor = this.highlightColor.getARGB();
+		List< Pair< String, Integer > > tagColors = new ArrayList<>( Collections.nCopies( 2 * n + 1, null ) );
+		tagColors.set( 0, Pair.of( "default", defaultColor ) );
+		for ( int i = 0; i < n; i++ )
+		{
+			int color = ColorUtils.mixColors( highlightColor, defaultColor, ( float ) i / n );
+			String label = "T-" + ( i + 1 );
+			tagColors.set( n - i, Pair.of( label, color ) );
+		}
+		for ( int i = 0; i < n; i++ )
```

Comment:
this is maybe unnecessarily the same - symmetrical, but it makes no sense to optimize it either

unless you maybe want to offer a transition from mother-default, over division, to daughters-new-default colors
(I could imagine color encoding a generation number.. but that's for some different story)